.github/workflows/build-plugin.yml

```diff
@@ -139,7 +139,7 @@ jobs:
     needs: set-variables
     concurrency:
       group: ci-${{ github.ref }}
-    if: ${{ needs.set-variables.outputs.package == 'framework-harmony' }}
+    if: ${{ needs.set-variables.outputs.package == 'framework-harmony' || needs.set-variables.outputs.package == 'framework-houdini'}}
```

Comment:
If the only change is 3.9, then the if should go in the python installation, so we avoid duplicated code.